entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z "${SONAR_TOKEN}" ]]; then
+  echo "This GitHub Action requires the SONAR_TOKEN env variable."
+  exit 1
+fi
+
+if [[ -z "${SONAR_HOST_URL}" ]]; then
+  echo "This GitHub Action requires the SONAR_HOST_URL env variable."
+  exit 1
+fi
+
+if [[ -f "${INPUT_PROJECTBASEDIR%/}pom.xml" ]]; then
```

Comment:
I'm wondering how necessary this is? We might want to do this in bitbucket pipes too to be consistent.